### PR TITLE
update Getting Started

### PR DESCRIPTION
**Pull request by sympatheticmoose**
Resolves follow ups from https://github.com/weaveworks/weave-gitops-docs/pull/147

Fixes https://github.com/weaveworks/weave-gitops-docs/issues/155

### CODE REVIEW

**Comment by sympatheticmoose** [comment]
@jrryjcksn This PR addresses your original comments on the Getting Started guide. 

@davidstauffer would appreciate your review on the introduction section in particular.

File: docs/getting-started.md [hunk 1]
```diff
@@ -26,11 +44,47 @@ sudo mv ./gitops /usr/local/bin/gitops
...
+
+### 2 - Fork the podinfo sample application repository
+
+We will be making changes to the sample application to show GitOps reconciliation in action. So we will first fork the podinfo sample repository.
+
+Go to [https://github.com/wego-example/podinfo-deploy](https://github.com/wego-example/podinfo-deploy) and fork the repository.
+
+![fork](/img/github-fork.png)
+
+`Podinfo` is simple, Go web application made up of a frontend and backend component; it is designed to showcase best practices of running microservices in Kubernetes. The full application source can be found [here](https://github.com/stefanprodan/podinfo).
```
Comment (JamWils) [inline docs/getting-started.md:69]:
```suggestion
`Podinfo` is a simple web application written in Go made up of a frontend and backend component; it is designed to showcase best practices of running microservices in Kubernetes. The full application source can be found [here](https://github.com/stefanprodan/podinfo).
```

**Review by JamWils — APPROVED** [review]
Just one suggestion.  This is a really nice set of changes and linking to all of the external resources is really helpful for those that want to dig deeper into Flux, etc.

File: docs/getting-started.md [hunk 2]
```diff
@@ -54,73 +108,101 @@ Have a nice day! 👋
...
+  - Specify the same configuration repository from steps 1 and 4 (install).
 - Path: **./**  
-  - This allows you to specify a particular folder with a repository, should the repo contain more than a single application's deployment manifests.
+  - **Leave as default**, this allows you to specify a particular folder with a repository, should the repo contain more than a single application's deployment manifests.
 - Branch: (**should not require changing**)
 
-Click **Submit** in the bottom right of the form. This will result in an error as we have not recently authenticated with GitHub, so we are not yet authorized to raise a pull request against your repository.
-
-Click **Authenticate with GitHub** within the error message as shown below:
+Click **AUTHENTICATE WITH GITHUB** next to the `Source Repo URL` field.
```
Comment (josecordaz) [inline docs/getting-started.md:203]: Small suggestion would be to add some dashes(or something similar) to help differentiate between user steps and info,  maybe like this:
image

**Review by jpellizzari — APPROVED** [review]

**Review by davidstauffer — COMMENTED** [review]
Great Improvement on the getting started guide.

**Pull request merged**